QUARKUS-3461.md

```diff
@@ -0,0 +1,40 @@
+# QUARKUS-3461 Verify behaviour and functionality of Quarkus CLI update command with RHBQ
+- JIRA link: https://issues.redhat.com/browse/QUARKUS-3461
+- Quarkus Guides:
+    - https://quarkus.io/guides/update-quarkus
+
+## Scope of the testing
+- Exploratory testing of the feature over supported versions of RHBQ and Quarkus upstream
+- Writing test coverage for this feature is continual work, requires update for every release of Quarkus
+- Test update between each of Quarkus LTS release versions
+- Test coverage in QQE TS:
+  - Test most frequently applied OpenRewrite recipes:
+    - org.openrewrite.properties.ChangePropertyKey
+    - org.openrewrite.quarkus.ChangeQuarkusPropertyKey
+    - org.openrewrite.java.ChangeMethodName
+    - org.openrewrite.java.dependencies.ChangeDependency
+    - org.openrewrite.maven.UpgradePluginVersion
+    - org.openrewrite.maven.UpgradeDependencyVersion
+    - org.openrewrite.quarkus.AddQuarkusProperty
+    - org.openrewrite.quarkus.DeleteQuarkusProperty
+    - org.openrewrite.maven.AddDependency
+  - Add static Quarkus applications, where each application should contain a group of dependencies and properties that were modified in the [recipes](https://github.com/quarkusio/quarkus-updates/tree/main/recipes/src/main/resources/quarkus-updates/core)
```

Comment:
Did you abandon dynamically created apps we talked about with the Quarkus CLI service? I am not sure if having full projects is better than something like: `cliClient.createApplication("app", new Extensions { QUARKUS_REST, SECURITY}, new Class[] { TestClass1.class, TestClass2.class }, "quarkus.oidc.enabled=false\n")`.

Reply:
For simple apps Quarkus CLI service is enough, but multi-module project cannot be created using CLI

Reply:
I can't think of an example of a test scenario where multi-module project is required and we are going to create it. I can see you addressed my comment so closing this thread.